Index: duckdb-0.6.1/setup.py
===================================================================
--- duckdb-0.6.1.orig/setup.py	2022-12-07 00:37:41.000000000 +0800
+++ duckdb-0.6.1/setup.py	2023-02-10 23:36:08.896180477 +0800
@@ -289,7 +289,7 @@
         'duckdb-stubs'
     ],
     include_package_data=True,
-    setup_requires=setup_requires + ["setuptools_scm<7.0.0", 'numpy>=1.14', 'pybind11>=2.6.0'],
+    setup_requires=setup_requires + ['numpy>=1.14', 'pybind11>=2.6.0'],
     use_scm_version = setuptools_scm_conf,
     tests_require=['google-cloud-storage', 'mypy', 'pytest'],
     classifiers = [
