Index: cupy-12.2.0/install/cupy_builder/install_build.py
===================================================================
--- cupy-12.2.0.orig/install/cupy_builder/install_build.py
+++ cupy-12.2.0/install/cupy_builder/install_build.py
@@ -151,7 +151,6 @@ def get_compiler_setting(ctx: Context, u
 
     if rocm_path:
         include_dirs.append(os.path.join(rocm_path, 'include'))
-        include_dirs.append(os.path.join(rocm_path, 'include', 'hip'))
         include_dirs.append(os.path.join(rocm_path, 'include', 'rocrand'))
         include_dirs.append(os.path.join(rocm_path, 'include', 'hiprand'))
         include_dirs.append(os.path.join(rocm_path, 'include', 'roctracer'))
@@ -160,7 +159,6 @@ def get_compiler_setting(ctx: Context, u
         include_dirs.append(os.path.join(rocm_path, 'include', 'hipfft'))
         include_dirs.append(os.path.join(rocm_path, 'include', 'rocsolver'))
         include_dirs.append(os.path.join(rocm_path, 'include', 'rccl'))
-        library_dirs.append(os.path.join(rocm_path, 'lib'))
 
     if use_hip:
         extra_compile_args.append('-std=c++11')
Index: cupy-12.2.0/install/cupy_builder/_compiler.py
===================================================================
--- cupy-12.2.0.orig/install/cupy_builder/_compiler.py
+++ cupy-12.2.0/install/cupy_builder/_compiler.py
@@ -236,7 +236,7 @@ class DeviceCompilerUnix(DeviceCompilerB
         compiler_so = rocm_path
 
         hip_version = build.get_hip_version()
-        postargs = ['-O2', '-fPIC', '--include', 'hip_runtime.h']
+        postargs = ['-O2', '-fPIC', '--include', 'hip/hip_runtime.h']
         if hip_version >= 402:
             postargs += ['--std=c++14']
         else:
