Index: cupy-10.0.0/install/cupy_builder/install_build.py
===================================================================
--- cupy-10.0.0.orig/install/cupy_builder/install_build.py
+++ cupy-10.0.0/install/cupy_builder/install_build.py
@@ -34,7 +34,7 @@ def _tempdir():
     try:
         yield temp_dir
     finally:
-        shutil.rmtree(temp_dir, ignore_errors=True)
+        pass
 
 
 def get_rocm_path():
