Index: cupy-11.0.0/cupy/_core/_routines_linalg.pyx
===================================================================
--- cupy-11.0.0.orig/cupy/_core/_routines_linalg.pyx
+++ cupy-11.0.0/cupy/_core/_routines_linalg.pyx
@@ -626,6 +626,16 @@ cpdef _ndarray_base tensordot_core(
     if _cuda_runtime_version < 0:
         _cuda_runtime_version = runtime.runtimeGetVersion()
 
+    if (
+        not runtime._is_hip_environment and
+        _cuda_runtime_version >= 11000 and
+        compute_capability >= 50
+    ):
+        tensordot_core_v11(transb, transa, m, n, k, b, ldb, a, lda, c, m)
+        if copy_to_out is not None:
+            elementwise_copy(copy_to_out, out)
+        return out
+
     handle = device.get_cublas_handle()
     if dtype == 'e':
         coef_dtype = 'f'
